src/main/java/de/zalando/aruha/nakadi/controller/EventTypeController.java

```diff
@@ -214,14 +212,7 @@ private void validateName(final String name, final EventType eventType) throws I
     }
 
     private void validateSchemaChange(final EventType eventType, final EventType existingEventType) throws InvalidEventTypeException {
-        final boolean existingNonNullSchemaChanged =
-                existingEventType.getSchema() != null && !existingEventType.getSchema().equals(eventType.getSchema());
-
-        final boolean existingNullSchemaChanged =
-                existingEventType.getSchema() == null && eventType.getSchema() != null;
-
-
-        if (existingNonNullSchemaChanged || existingNullSchemaChanged) {
+        if (!existingEventType.getSchema().equals(eventType.getSchema())) {
```

Comment:
This could result in `NullPointerException` since there could be existing event types with `null` schema on sandbox. Maybe we should have a "release notes" where we can keep track of such information in order to run "normalizing" routines after deploying.